client/src/pages/AdminLogin/index.tsx

```diff
@@ -1,5 +1,99 @@
+import { Button, TextField, Typography } from '@mui/material';
+import { useFormik } from 'formik';
+import { useContext, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { UserContext } from '../../context';
+import { AdminLoginSchema } from '../../helpers/validationSchema';
+import { UserContextTypeWithDispatch } from '../../interfaces';
+import httpInstance from '../../services';
+import './styles.css';
+
 function AdminLogin() {
-  return (<h1>AdminLogin</h1>);
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [responseError, setResponseError] = useState<string>('');
+  const { setUserInfo }:UserContextTypeWithDispatch = useContext(UserContext);
+  const navigate = useNavigate();
+  const formik = useFormik({
+    initialValues: {
+      username: '',
+      password: '',
+    },
+    validationSchema: AdminLoginSchema,
+    onSubmit: (values) => {
+      const Admin = async () => {
```

Comment:
```suggestion
      const Admin = async () => {
```
Not sure what function name means?